src/AdminTab/TableList.tsx

```diff
@@ -41,9 +41,17 @@ const TableDropdown: FunctionComponent<TableDropdownProps> = ({ table, onOpenMet
       >
         {t('New SQL query')}
       </Menu.Item>
-      <Menu.Item icon={<DeleteOutlined />} key={`${table.name}-remove`} onClick={() => removeTable(table.name)}>
-        {t('Remove')}
-      </Menu.Item>
+      <Popconfirm
```

Comment:
I just remembered (late :( ) I think Edon had the idea to implement an ohnosecond "Undo" button, showing up in the notification popup, somewhat like what you have in Gmail.

Another (late as well) thought is to maybe separate the ":wastebasket: Remove" separated by a separator in the menu, so it's harder to misclick?

Reply:
Hm, I'm not sure what to do here... Do you want me to add a separator now? There are only two items in the menu, not sure if it makes sense...

Reply:
Sorry about being vague here. I'm just tossing alternative ideas, but I don't have a strong opinion. Also the "Undo" thing is much more work. So if neither appeals to you now, let's just leave them for later and merge as is.